Examples/Input_Customfunction_Nestedresult.json

```diff
@@ -0,0 +1,94 @@
+﻿{
```

Comment:
This file will be better off in folder UnitTests/Inputs, there's already some input files there, and it avoids the ugly path in CustomFunctionsTest file.

Reply:
Got it, I just moved to Inputs in tests the files and changed the paths.